ionic_user_interface/src/components/gym-selector/GymSelector.vue

```diff
@@ -72,19 +87,23 @@ import {
   IonSelectOption,
   IonRow,
   IonCol,
-  IonButton,
 } from '@ionic/vue';
 import Lookup, { Country } from 'country-code-lookup';
-import { map, mapOutline, warning } from 'ionicons/icons';
+import { map, mapOutline, warning, locateOutline } from 'ionicons/icons';
 
 import MessageBox from '@/components/MessageBox.vue';
+import LoadingButton from '@/components/LoadingButton.vue';
 import GymMap from '@/components/GymMap.vue';
-import getGymsByCountry, { GymLocation } from '@/common/api/route/getGymsByCountry';
+import getGymsByCountry, { LatLong, GymLocation } from '@/common/api/route/getGymsByCountry';
 import AutoComplete from './AutoComplete.vue';
+import { Geolocation } from '@capacitor/geolocation';
+import axios from 'axios';
+import haversine from 'haversine';
```

Comment:
External dependencies help me move on top

Reply:
Is there a way to do this automatically using prettier?

Reply:
Also, it would be nice for multiple imports from the same source to be alphabetically arranged

Reply:
hmm i can try nxt time with this https://github.com/simonhaenisch/prettier-plugin-organize-imports#readme

Reply:
wait no it doesnt separate internal external.  this one does but vue not supported yet :( https://github.com/trivago/prettier-plugin-sort-imports

Reply:
how about eslint?

Reply:
can sort. but still need to manually separate internal external. i think no point

Reply:
Then we should just not separate into internal external. Having a sorting order is enough

Reply:
wah i prefer clear line between internal and external instead of sorting. ok we can make it issue

Reply:
#152

Reply:
I also prefer that but without a linter the benefits (which are not much except nicer looking imports) aren't outweighed by the time/effort cost